src/_cffi_src/openssl/ssl.py

```diff
@@ -594,6 +614,10 @@
 long (*DTLS_get_link_min_mtu)(SSL *) = NULL;
 #endif
 
+#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_111
+size_t (*DTLS_get_data_mtu)(SSL *) = NULL;
```

Comment:
Take a look in `_conditional.py` to see what we do when a symbol isn't actually available.

Reply:
It looks like currently, `_conditional.py` doesn't actually remove any conditionally-available functions (e.g. `DTLS_get_link_min_mtu`, visible just above my new code in the diff, doesn't appear there). So I made my best guess at how it should work for `DTLS_get_data_mtu` -- take a look.

Reply:
Ha, I would argue that is a bug (we don't want callables that are null func ptrs!). Looks like that's a thing we need to separately fix for the DTLS bindings though...